massa-models/src/denunciation.rs

```diff
@@ -739,6 +739,16 @@ pub enum DenunciationIndex {
     },
 }
 
+impl DenunciationIndex {
+    /// Get field: slot
+    pub fn get_slot(&self) -> &Slot {
+        match self {
+            DenunciationIndex::BlockHeader { slot } => slot,
+            DenunciationIndex::Endorsement { slot, .. } => slot,
+        }
+    }
+}
+
 impl PreHashed for DenunciationIndex {}
```

Comment:
Warning: DenunciationIndex is NOT prehashed. This is a security risk.

Reply:
Fixed